stdlib/private/OSLog/OSLogMessage.swift

```diff
@@ -21,6 +21,7 @@
 ///
 /// See `OSLogInterpolation.appendInterpolation` definitions for default options
 /// for integer types.
+@_frozen
```

Comment:
This is dangerous. I can see this enum being extended. The very first thing that comes to mind is `binary`. Does it have to be `@_frozen`?

Reply:
I will try to see if we an avoid making this frozen. There were two reasons for doing this IIRC:
* I think the main reason was that there were some SIL instructions not supported by the constant evaluator when the enum was made resilient, as reads and switching etc happen through addresses. This should be fixable I guess. 
* I think @unknown default had to be handled in that case. 

Btw, `binary` and other standard printf options should be in this enum before this goes public. Also, making this @frozen or not should not affect adding custom formatting options for integers, which can be accomplished by adding new overloads of `appendInterpolation` that takes an `Int` and new enum like `DarwinIntFormat` that will specify the new custom options.